### ISSUE TITLE
use local storage for data

### ISSUE THREAD

**Issue by alexweininger** [alexweininger/recent-pdf#58]
This would make load times faster and would provide a much better user exp. bc right now pdf files need to be used pretty recently for them to show up in the list.

**Comment by JuliaFC** [comment]
Can I also take on this as well?

**Comment by alexweininger** [comment]
@JuliaFC Yes. I would like to talk about how you want to implement it though :)

**Comment by JuliaFC** [comment]
@alexweininger absolutely! As soon as I read through the code and study the best way to do it, I'd be happy to talk to you about it! Or is there anything you'd like to discuss right away?

**Comment by alexweininger** [comment]
@JuliaFC Nothing to discuss right away.
